fpga/src/main/scala/arty100t/Configs.scala

```diff
@@ -26,6 +26,12 @@ class WithArty100TTweaks extends Config(
   new WithArty100TUARTTSI ++
   new WithArty100TDDRTL ++
   new WithNoDesignKey ++
+  new chipyard.harness.WithHarnessBinderClockFreqMHz(50) ++
+  new chipyard.config.WithMemoryBusFrequency(50.0) ++
+  new chipyard.config.WithSystemBusFrequency(50.0) ++
+  new chipyard.config.WithPeripheryBusFrequency(50.0) ++
```

Comment:
> For FPGA prototypes, can you do multi-clock designs? Or are you enforced to have a single frequency across all clock domains? If that is the case, is there some require/assert to enforce this?

The existing FPGA designs are all single clock. To support multi-clock FPGA, a `HarnessClockInstantiator` variant would have to be designed that would configure the clock generator IP for that particular board. That is future work for the next use case which wants multi-clock-target FPGA prototypes.

The broadcast-single-clock approach is the simpler thing to do, and hopefully it can make it more clear how one might bring up a new board. The `AllClocksFromHarnessClockInstantiator` will check/enforce that all requested clocks match the reference clock frequency.

> Why should the harness clock have a specific value? Can't you just choose the default fastest clock of the system and use that?

Its not clear to me that using the fastest clock in the system is the most obvious behavior. On a realistic system, where the tile runs faster than the uncore, this doesn't really make sense.

Its also not clear to me that using the slowest clock is the most intuitive behvior either.

I feel like the most intuitive, and realistic, behavior, is for harness clocks to be completely async/unrelated with chip clocks. 
A more aggressive refactor of the HarnessBinders could remove the `harnessBinderClock` entirely, and force all HarnessBinders to request their own clock at whatever frequency they need, instead of relying on some implicit clock.